php-checks/src/main/java/org/sonar/php/checks/formatting/FunctionSpacingCheck.java

```diff
@@ -44,14 +44,14 @@ private void checkClosureSpacing(FormattingStandardCheck formattingCheck, AstNod
     Token lParenToken = node.getFirstChild(PHPPunctuator.LPARENTHESIS).getToken();
     Token functionKeyword = node.getFirstChild(PHPKeyword.FUNCTION).getToken();
 
-    if (getNbSpaceBetween(functionKeyword, lParenToken) != 1) {
-      formattingCheck.reportIssue("Put exactly one space between the \"function\" keyword and the opening parenthesis.", node);
-    }
+//    if (getNbSpaceBetween(functionKeyword, lParenToken) != 1) {
+//      formattingCheck.reportIssue("Put exactly one space between the \"function\" keyword and the opening parenthesis.", node);
+//    }
 
     AstNode lexicalVars = node.getFirstChild(PHPGrammar.LEXICAL_VARS);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "lexicalVars" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1481)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this useless assignment to local variable "lexicalVars". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1854)